### PR TITLE
build: Update packaging and tests

### PR DESCRIPTION
**Pull request by MRigal**

### CODE REVIEW

File: .github/workflows/test_and_coverage.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        pip install tox tox-gh-actions
+        pip install -r example/requirements.txt
+    - name: Run Tox
+      run: tox
+    - name: Report to coverall  # by the package as the action is broken, but won't work on external forks!
+      if: matrix.python-version == 3.7
+      env:
+        COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+      run: |
+        pip install coveralls
```
Comment (anrie) [inline .github/workflows/test_and_coverage.yml:34]: I wouldn't mind removing coveralls completely and switch to a better integrated solution like codecov in the future.

File: README.rst [hunk 2]
```diff
@@ -13,6 +13,9 @@ django-image-cropping
     :target: https://coveralls.io/r/jonasundderwolf/django-image-cropping
```
Comment (anrie) [inline README.rst:13]: This badge is obsolete in case we drop coveralls.

File: tox.ini [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
+    3.5: py35
+    3.6: py36
+    3.7: py37
+    3.8: py38
 
 [testenv]
-commands = py.test example --create-db --cache-clear
+commands =
+    pytest example --create-db --cache-clear --cov-append
+;   # TODO: fixme it is empty and thus failing
```
Comment (anrie) [inline tox.ini:21]: This could also be completely removed.

File: README.rst (hunk 2 above)
Comment (MRigal) [reply README.rst]: done

File: .github/workflows/test_and_coverage.yml (hunk 1 above)
Comment (MRigal) [reply .github/workflows/test_and_coverage.yml]: commented out for now

**Pull request merged**